README.md

```diff
@@ -2,13 +2,9 @@
 These are the setup scripts required to fill data into the internal database.
 
 ## Main scripts
-### sample.py
-* What does it do?
+### voting_rights/voting_rights.py
+Parses our compilation of voting laws from voting_rights/voting_rights_2017.csv and outputs SQL file voting_rights/voting_rights.sql which can be executed to truncate and repopulate the voting_rights MySQL table.
 
```

Comment:
i hate this feature of github..they don't port comments between commits.
I think for now we can leave the voting_year to be just the year..I doubt we'll get granular data within a year. Also do you need to rename the csv here to be _2016 in the comment?